Oqtane.Server/Infrastructure/Jobs/NotificationJob.cs

```diff
@@ -186,7 +186,7 @@ await client.AuthenticateAsync(settingRepository.GetSettingValue(settings, "SMTP
                                     var mailboxAddressValidationError = "";
 
                                     // sender
-                                    if ((settingRepository.GetSettingValue(settings, "SMTPRelay", "False") == "True") && string.IsNullOrEmpty(fromEmail))
+                                    if ((settingRepository.GetSettingValue(settings, "SMTPRelay", "False") != "True") || (settingRepository.GetSettingValue(settings, "SMTPRelay", "False") == "True") && string.IsNullOrEmpty(fromEmail))
```

Comment:
Looks like the bracket put in wrong place, IMO this can be changed to:
```
if (settingRepository.GetSettingValue(settings, "SMTPRelay", "False") != "True" || string.IsNullOrEmpty(fromEmail))
```